standalone-node/provisioning_scripts/install-os.sh

```diff
@@ -701,6 +708,28 @@ EOT
 
 }
 
+# Copy user-apps to OS disk under /opt
```

Comment:
- No error handling if mount fails.
- Uses /mnt2, which may not exist or may already be in use.
- Uses cp -r, which can fail silently on permission errors or partial copies.